lambdas/services/document_service.py

```diff
@@ -121,3 +121,21 @@ def delete_documents(
             )
 
             self.update_item(table_name, reference.id, updated_fields=update_fields)
+
+    def delete_documents_by_type(
+        self,
+        doc_type: str,
+        document_references: list[DocumentReference],
+        type_of_delete: str,
+    ):
+        table_name = ""
+        if doc_type == SupportedDocumentTypes.ARF.value:
+            table_name = os.environ["DOCUMENT_STORE_DYNAMODB_NAME"]
+        elif doc_type == SupportedDocumentTypes.LG.value:
+            table_name = os.environ["LLOYD_GEORGE_DYNAMODB_NAME"]
+
+        return self.delete_documents(
```

Comment:
If table name is empty we shouldn't call this.

Reply:
In the other methods in DocumentService, I don't see a pattern of special handling when table_name or doc_type is empty. 
When we run delete_document with table_name empty, boto3 would raise a client error, which will be handled by the lambda handler.

Reply:
I don't think we should get to boto3 with invalid request, even if it would raise ClientError. If the other methods in the service has the same issue I assume it would be refactor as part of another ticket?

Reply:
I have changed the implementation, now table_name shouldn't be empty in any case.